stdlib/public/Darwin/Accelerate/vImage_Buffer.swift

```diff
@@ -0,0 +1,348 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Accelerate
+
+//===----------------------------------------------------------------------===//
+//
+//  vImage_Buffer
+//
+//===----------------------------------------------------------------------===//
+
+@available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+extension vImage_Buffer {
+    
+    /// Returns the size, in pixels, of a vImage buffer.
+    public var size: CGSize {
```

Comment:
Why do we want to expose a `size` property instead of the existing `.width` and `.height`? What's the intended use case for this?

Reply:
As per the header. I'm exposing the various free functions as computed properties.

 *  Rationale: If you attempt your own home-made conversion to CGSize / NSSize by
 *            ordinary C rules and the value rounds, it will round up half the time.
 *            This could lead to a crash later because the height or width
 *            will be reported to be larger than it really is and an ensuing image
 *            operation will attempt to touch scanlines that don't exist.

Reply:
Ok, then please update the comment to be clear that the size is *rounded down* and you may lose part of your image, especially on 32b systems.

Reply:
done!